packages/generator/templates/app/src/app/layout.tsx

```diff
@@ -0,0 +1,23 @@
+import "./styles/globals.css"
+import {BlitzProvider} from "./blitz-client"
+import {Inter} from "next/font/google"
+
+const inter = Inter({subsets: ["latin"]})
+
+export const metadata = {
+  title: "New Blitz App",
```

Comment:
```suggestion
  title: { title: "New Blitz App", template: "%s – Blitz" }
```

I think we can start educating people about the metadata API a bit more with this. 
Docs at https://nextjs.org/docs/app/api-reference/functions/generate-metadata#template